stregsystem/templates/admin/stregsystem/report/ranks.html

```diff
@@ -134,5 +134,6 @@ <h1>Rangeringer for {{year}}</h1>
 </div>
 <div style="clear: both;">&nbsp;</div>
 <a href="./{{last_year}}">Forrige år</a>
+{% if show_next_year %}<a style="float: right;" href="./{{next_year}}">Næste år</a>	{% endif %}

```

Comment:
There's a weird space here.
```suggestion
{% if show_next_year %}<a style="float: right;" href="./{{next_year}}">Næste år</a>{% endif %}
```